lib/metadata/util/win32/Win32Software.rb

```diff
@@ -124,7 +125,7 @@ def registry_applications_user_data(reg_node)
       end
     end
 
-    def registry_applications_app_paths(registry_doc, registry_path)
+    def registry_applications_app_paths(registry_doc, registry_path, fs)
       return unless (reg_node = MIQRexml.findRegElement(registry_path, registry_doc.root))
       postProcessApps(reg_node, fs)
```

Comment:
@hsong-rh How did this ever work in the past? Do we know if there was a recent change that broke this?

Reply:
Yes, it's broke by our recent refactor change https://github.com/ManageIQ/manageiq-smartstate/pull/71